lib/nimble_template/addons/variants/phoenix/api/error_view.ex

```diff
@@ -0,0 +1,46 @@
+defmodule NimbleTemplate.Addons.Phoenix.Api.ErrorView do
+  @moduledoc false
+
+  use NimbleTemplate.Addons.Addon
+
+  @impl true
+  def do_apply(%Project{} = project, _opts) do
+    project
+    |> delete_files()
+    |> copy_files()
+  end
+
+  defp copy_files(
+         %Project{
+           base_module: base_module,
+           web_module: web_module,
+           web_path: web_path,
+           web_test_path: web_test_path
+         } = project
+       ) do
+    binding = [
+      web_module: web_module,
+      base_module: base_module
+    ]
+
+    files = [
+      {:eex, "lib/otp_app_web/views/error_helpers.ex.eex", "#{web_path}/views/error_helpers.ex"},
+      {:eex, "lib/otp_app_web/views/api/error_view.ex.eex", "#{web_path}/views/api/error_view.ex"},
+      {:eex, "test/otp_app_web/views/error_helpers_test.exs",
+       "#{web_test_path}/views/error_helpers_test.exs"},
+      {:eex, "test/otp_app_web/views/api/error_view_test.exs",
+       "#{web_test_path}/views/api/error_view_test.exs"},
+      {:eex, "test/support/view_case.ex.eex", "test/support/view_case.ex"}
+    ]
+
+    Generator.copy_file(files, binding)
+
+    project
+  end
+
+  defp delete_files(%Project{web_path: web_path} = project) do
```

Comment:
I'm thinking should we place this function before the `copy_file/1` so it is easier to see what happens first 🤔 What do you think?

Reply:
yeah, updated in https://github.com/nimblehq/elixir-templates/pull/140/commits/e355bb4b1264d6ff57a02f3ebf736394b52781d3